scripts/powershell/sub-cleanup/sub-cleanup.ps1

```diff
@@ -0,0 +1,70 @@
+﻿# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Requires PowerShell modules Az and RemoteSigned excecution policy
+#
+# Commands to make this happen:
+#   Install-Module -Name Az -AllowClobber
+#   Set-ExecutionPolicy RemoteSigned
+
+function Print-Help 
```

Comment:
FYI see https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help?view=powershell-7 for information about how to document the help text so that Powershell can understand it and provide tooling for it.

Reply:
It also provides a consistent way of getting help in PowerShell.